Formula/neomutt.rb

```diff
@@ -1,8 +1,8 @@
 class Neomutt < Formula
   desc "E-mail reader with support for Notmuch, NNTP and much more"
   homepage "https://neomutt.org/"
-  url "https://github.com/neomutt/neomutt/archive/20211022.tar.gz"
-  sha256 "49aa5029665c6819e708276b9efa1ca71ec5afe870eb9f08e656107d234941e6"
+  url "https://github.com/neomutt/neomutt/archive/refs/tags/20211029.tar.gz"
```

Comment:
```suggestion
  url "https://github.com/neomutt/neomutt/archive/20211029.tar.gz"
```
Do we really need the longer URL? They resolve to the same place. (See my previous suggestion.)

Reply:
True, it seems that the shorter URL is enough.